lib/models/instrumentation.js

```diff
@@ -9,9 +9,9 @@ const logger = require('heimdalljs-logger')('ember-cli:instrumentation');
 let vizEnabled = utilsInstrumentation.vizEnabled;
 let instrumentationEnabled = utilsInstrumentation.instrumentationEnabled;
 
-function _enableFSMonitorIfInstrumentationEnabled() {
+function _enableFSMonitorIfInstrumentationEnabled(config) {
```

Comment:
Need to be able to pass the `config` in here so we can "mock" it in tests? Is this a smell or okay to do?